CONTRIBUTING.md

```diff
@@ -1,193 +1,49 @@
-[developer guides](https://probnum.readthedocs.io/en/latest/development/developer_guides.html)
-for detailed instructions.
+If you would like to contribute but are unsure how, then writing turorials, documentation or working on
+[open issues](https://github.com/probabilistic-numerics/probnum/issues) are a good way to start. 
 
-## Getting Started
+## Reporting a bug
+If you find a bug, please report it by opening an issue on the GitHub page, and tag it as a bug.
+Reproducibility is key, so if possible provide a working code snipped that produces the bug, and tell us the python 
+version and ProbNum version you used.
 
-Begin by forking the repository on GitHub and cloning your fork to a local machine.
+Please also detail how you found the bug, what you expected to happen, and what happened instead. Have you made any 
+code changes after which the bug occurred?
 
-```shell
-git clone git@github.com:MyGithubAccount/probnum.git
-```
+Of course, bug-fixes via pull request are very welcome as well.
 
-Next, create a new branch in your
-forked repository describing the feature you would like to implement.
-```shell
-git checkout -b my-new-feature
-```
-You can now get started writing code in your new
-branch. Make sure to keep the following best practices regarding code quality in mind.
+## Requesting a feature
+If you are working with ProbNum, and you are missing a feature, please open an issue and detail what feature you 
+would like to see. Please also check existing issues in case someone else requested the same or a similar feature. 
+Detail the indented functionality and use case of the feature. 
+You can also lay out how you would implement the feature. 
+If you have a working version, please consider doing a pull request!
 
-### Code Quality
+## Contributing via pull request
 
-Code quality is an essential component in a collaborative open-source project.
+You implemented an additional feature, a tutorial, tests, or some other enhancement to ProbNum? That's great! 
+Please consider doing a pull request (PR) of the changes. 
 
-- Make sure to observe [good coding practice](https://www.python.org/dev/peps/pep-0020/).
-- Keep dependencies to a minimum.
-- All code should be covered by tests within the [pytest](https://docs.pytest.org/) framework.
-- Documentation of code is essential. ProbNum uses the
-[NumPy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html).
-- Code should be formatted with [*Black*](https://github.com/psf/black) and follow the internal [style guide](https://github.com/probabilistic-numerics/probnum/blob/master/STYLEGUIDE.md).
-  For more thorough Python code style guides we refer to [PEP 8](https://www.python.org/dev/peps/pep-0008/) and to [the *Black* code style](https://github.com/psf/black/blob/master/docs/the_black_code_style.md).
+If you do so, please check out the [developers guide](development/contributing_code) first.
```

Comment:
"developer guide"

the link does not point to a page (see https://probnum--414.org.readthedocs.build/en/414/development/development/contributing_code)

Reply:
typo fixed. 
Concerning the link, how would I link from a markdown file to another (not a sphinx expert here)? I am especially puzzled since this link can be clicked from RTD as well as from the MD file on the github page.

Reply:
try `contributing_code` or `../development/contributing_code`. The path this points to is `/development/development/contributing_code`:  https://probnum--414.org.readthedocs.build/en/414/development/development/contributing_code

Reply:
I tried both, and it does not work. It works when I use `contributing_code.html`.

Then again, even if that link works on RTD, it won't work if someone clicks on the link when opening the markdown from the GitHub page directly. Not sure what to do there.

Reply:
I added an absolute path to RTD for now. That is a bit ugly, but at least one can access the page from both RTD and GitHub that way. Happy to change it if there is a better way.

Reply:
Ah I remember this being an issue now. I also used absolute links in the `README.md`.